rebar.config

```diff
@@ -78,8 +78,7 @@
     {include_erts, false},
     {sys_config, "erldns.config"},
     {overlay, [
-        {copy, "priv/zones-example.json", "priv/zones-example.json"},
-        {copy, "priv/zones-test.json", "priv/zones-test.json"}
+        {copy, "priv/zones/", "priv/zones"}
```

Comment:
Is this line required?

Reply:
Unfortunately, if we want the dir to be loaded to release builds 🥲